notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
<p>Bad link in automl<strong>//</strong>automl_image_object_detection_online_prediction.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>